### PR TITLE
Fix robottest.Translation

### PR DESCRIPTION
**Pull request by vanzaj**
Test of the first translation was failing because it was completed
before the image was loaded.

### CODE REVIEW

File: robottests/translation.txt [hunk 1]
```diff
@@ -29,6 +29,7 @@ Can switch to other language  [Arguments]  ${button}  ${language}
 Enter Translation
     Open Browser  ${BASE_URL}  ${BROWSER}
     Click Link  Translations
+    Set Browser Implicit Wait     0.2 seconds
```
Comment (basvodde) [inline robottests/translation.txt]: What is an implicit wait? Is it possible to change this to a polling and not a sleeping?

Comment (vanzaj) [inline robottests/translation.txt]: It's the only way I found to make browser sleep for a bit while the image gets loaded. This adds ~1 sec to overall tests.

Comment (basvodde) [inline robottests/translation.txt]: Yup, but sleeps are unreliable and make tests slow, so you'll want to poll instead. In Robot, there should be Wait Until keywords which poll. Then you can wait until image is loaded, with a timeout. That ought to be better.

**Comment by basvodde** [comment]
Getting there, Travis still not fixed :(

**Pull request merged**